packages/atomic/src/components/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Component,
+  h,
+  Element,
+  State,
+  Prop,
+  Listen,
+  Method,
+} from '@stencil/core';
+import {
+  ResultTemplatesManager,
+  ResultsPerPageState,
+  ResultsPerPage,
+  buildFoldedResultList,
+  FoldedResultList,
+  FoldedResult,
+  FoldedResultListState,
+  FoldedResultListProps,
+  SearchEngine,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {
+  ResultDisplayDensity,
+  ResultDisplayImageSize,
+} from '../../atomic-result/atomic-result-display-options';
+import {TemplateContent} from '../../result-template/atomic-result-template/atomic-result-template';
+import {ResultList} from '../result-list';
+import {
+  getTemplate,
+  handleInfiniteScroll,
+  initializeResultList,
+  postRenderCleanUp,
+} from '../result-list-common';
+
+/**
+ * The `atomic-folded-result-list` component is responsible for displaying query results by applying one or more result templates.
+ *
+ * @internal
+ * @part result-list - The element containing every result of a result list
+ */
+@Component({
+  tag: 'atomic-folded-result-list',
+  styleUrl: 'atomic-folded-result-list.pcss',
```

Comment:
I would extract all the contents of `atomic-result-list.pcss` into a `atomic-common-result-list.pcss` file or something similar, and I would make both `atomic-folded-result-list.pcss` and `atomic-result-list.pcss` extend it.

This way, when we make changed to `atomic-common-result-list.pcss`, we know we're affecting more than one component.